README.md

```diff
@@ -3,5 +3,33 @@
 ### Welcome to my first setup and mobile Portfolio!
 
 
-*_let me know what you think_*👌
+Built With
+CSS
+HTML
 
+Authors
+👤 Susana González Gallardo
+
+GitHub: @Zuz4n
+
+Twitter: @Suschen
+
+LinkedIn: siuxmicroverse@gmail.com
+
+
+*_🤝 Contributing_*
+*_Contributions, issues, and feature requests are welcome!_*
+
+Feel free to check the issues page.
+
+Show your support
+*_Give a ⭐️ if you like this project!_*
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+
+📝 License
+
+This project is MIT licensed.
```

Comment:
- [ ] [OPTIONAL] I marked this as optional because the previous reviewers forgot to require it, please use the [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) for the Readme file, and adapt it with the information of your project.